joint_trajectory_controller/CMakeLists.txt

```diff
@@ -91,6 +92,7 @@ if(BUILD_TESTING)
     controller_manager
     realtime_tools
     ros2_control_test_assets
+    control_toolbox
```

Comment:
I am very sure that this is not needed. Is `control_toolbox` used explicitly in the test?

Reply:
Both tests gave me compile errors without including control_toolbox because the tests include test_trajectory_controller_utils.hpp, which includes joint_trajectory_controller.hpp, which includes control_toolbox/pid.hpp. Is there a way to have the includes from joint_trajectory_controller.hpp be included in the tests automatically with CMake?

Reply:
I guess this is related to https://github.com/ros-controls/ros2_controllers/pull/309#issuecomment-1065910124 ?